sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md

```diff
@@ -107,14 +107,14 @@ await foreach (BlobChangeFeedEvent changeFeedEvent in changeFeedClient.GetChange
 ## Troubleshooting
 All Blob service operations will throw a
 [RequestFailedException][RequestFailedException] on failure with
-helpful [`ErrorCode`s][error_codes].  Many of these errors are recoverable.
+helpful [`ErrorCode`][error_codes].  Many of these errors are recoverable.
```

Comment:
As Ted pointed out [here](https://github.com/Azure/azure-sdk-for-net/issues/13633#issuecomment-668761384), the s should still be here.